docs/source/developers/java/building.rst

```diff
@@ -272,6 +271,9 @@ CMake
           -DCMAKE_INSTALL_LIBDIR=lib/x86_64 ^
           -DCMAKE_INSTALL_PREFIX=java-dist ^
           -DCMAKE_UNITY_BUILD=ON ^
+          -DPARQUET_BUILD_EXAMPLES=OFF ^
+          -DPARQUET_BUILD_EXECUTABLES=OFF ^
+          -DPARQUET_REQUIRE_ENCRYPTION=OFF ^
```

Comment:
FYI: They are not harmful but they are redundant because they are `OFF` by default.